### PR TITLE
Feature weatherjs

### PR DESCRIPTION
**Pull request by billodiallo**
The page is made with modular valinna JS (JS6) and the following milestones are finished:
-  Setup HTML webpack to load style files, and pictures.
 - Setup Unsplash API and OpenWeather API to fetch data.
- Add extra spinning loader.
- styling 
- Use Eslint and stylint.

### CODE REVIEW

File: src/modules/dom.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  ${Math.round(data.feelsLike)}°C
+  `;
+  moreAbt[1].innerHTML = `
+  <i class='fa fa-temperature-high'></i>
+  <span class='b-feel'>Max temp</span>${Math.round(
+    data.tempMax,
+  )}°C, <i class='fa fa-temperature-low'></i>
+  <span class='b-feel'>Min temp</span>${Math.round(data.tempMin)}°C
+  `;
+  d.style.display = 'block';
```
Comment (rindrajosia) [inline src/modules/dom.js]: Here for example you can create a `CSS Class` called ` .block`  in your `style.css` then you can add to `d`

File: src/modules/dom.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  d.addEventListener('click', () => {
+    toggleTempF(d, c, data.temp);
+  });
+  displayCity.className = 'show';
+};
+
+export const selectImage = () => {
+  const script = document.createElement('script');
+  document.head.appendChild(script);
+  window.abc = function abc(data) {
```
Comment (rindrajosia) [inline src/modules/dom.js]: Please use arrow function

File: src/modules/dom.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    const body = document.querySelector('body');
+    body.style.background = `url('${data.items[2].media.m}') no-repeat`;
+    body.style.backgroundSize = 'cover';
```
Comment (rindrajosia) [inline src/modules/dom.js]: Keep all the styling logic in CSS files

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- [x] Good job by adding Necessary run common commands :+1:
- [x] Webpack is well configured :100: 

To improve :factory_worker: [REQUIRED]:

- [x] As your readme file is the first thing that users check, I suggest that you use tools like grammarly in order to help you with your grammar.
- [x] Your `live demo` and `issue` links are broken, It will be nice if you fix it.
- [x] Your APP provides three different temperatures. Please precise which one is correct or add a label for each one.


- [x] Please, keep all the styling logic in CSS files. (If you need to change something like a background image, add the classes with the different backgrounds to the CSS and then change the class name attribute with JavaScript) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/styling_requirement.png) from Pathwright)
- [x] As you are using ES6 syntax, please use `arrow functions` instead of the `function keyword `([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

Nice to have

- [x]  it has a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [x]  it fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Comment by billodiallo** [comment]
Hi, TSEs,
change done
Thanks for the review.

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED✔✔✔✔**

Hello @billodiallo 
Great work making the required changes.

You can merge👍👌

Happy coding 💻💻💻

**Pull request merged**